build.include

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
if editor is performing incorrect saving I would say that it's a bug of the editor, not of this file that could be kept clean.

Also as we only need user executable flag, we should only perform `chmod u+x`and not touch group or other permissions